### PR TITLE
[IMP] Master [Employee] Action menu (form & list)

### PR DESCRIPTION
**Pull request by Flotchet**
This Pr unclog the different hr action menu an move functionnalities in a more appropriate place to be more user firendly

task-3927336

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/report/hr_leave_employee_type_report.xml [hunk 1]
```diff
@@ -32,7 +32,6 @@
     <record id="action_hr_holidays_by_employee_and_type_report" model="ir.actions.server">
         <field name="name">Time off Analysis by Employee and Time Off Type</field>
         <field name="model_id" ref="hr_holidays.model_hr_leave_employee_type_report"/>
-        <field name="binding_model_id" ref="hr.model_hr_employee"/>
         <field name="state">code</field>
         <field name="groups_id" eval="[(4, ref('base.group_no_one'))]"/>
```
Comment (mepe-odoo) [inline addons/hr_holidays/report/hr_leave_employee_type_report.xml:36]: I think you can remove the groups :)

**Review by mepe-odoo — APPROVED** [review]
LGTM Just don't forgot to remove this line ;)

File: addons/hr_holidays/report/hr_leave_employee_type_report.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def init(self):
 
     @api.model
     def action_time_off_analysis(self):
-        domain = []
+        domain = [('company_id', 'in', self.env.context.get('allowed_company_ids', []))]
```
Comment (tivisse) [inline addons/hr_holidays/report/hr_leave_employee_type_report.py:124]:
```suggestion
        domain = [('company_id', 'in', self.env.companies.ids)]
```

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo rebase-ff

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**